sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationSettingTests.cs

```diff
@@ -116,7 +115,7 @@ public void SettingSomeFields()
 
             ConfigurationClient.BuildBatchQuery(builder, selector, null);
 
-            Assert.AreEqual($"http://localhost/?key=key&$select=key%2C%20value", builder.ToUri().AbsoluteUri);
+            Assert.AreEqual($"http://localhost/?key=key&$select=key%2Cvalue", builder.ToUri().AbsoluteUri);
```

Comment:
After the changes introduced in this PR, the output does not include empty spaces between field names:
```
Before changes: "select=key, value"
After changes: "select=key,value"
```
This doesn't change the service's behavior so I decided to not worry about it.